index.js

```diff
@@ -159,9 +161,9 @@ VueBuilderPlugin.prototype.apply = (compiler) => {
   compiler.plugin('watch-run', (compilation, callback) => buildVues(callback, compiler));
 
   compiler.plugin('after-compile', (compilation, callback) => {
-    // eslint-disable-next-line no-param-reassign
     compilation.fileDependencies = Array.from(compilation.fileDependencies).filter((file) => {
-      if (file.slice(-4) === '.vue') {
+
```

Comment:
Why is there an empty line?

Reply:
Not sure if I haveunderstood well, is now ok?